python/gtsam/utils/visual_data_generator.py

```diff
@@ -36,7 +36,7 @@ def __init__(self, K=Cal3_S2(), nrCameras: int = 3, nrPoints: int = 4) -> None:
         self.cameras = [Pose3()] * nrCameras
         self.points = [Point3(0, 0, 0)] * nrPoints
 
-    def print_(self, s="") -> None:
+    def print(self, s="") -> None:
```

Comment:
Looks great. nit: type hint missing here for `s`

Reply:
Done